Assets/_Project/Scripts/LevelEditor/TilePainter.cs

```diff
@@ -69,101 +72,101 @@ private void RegisterEditorSOs()
             {
                 // @todo fix moving platforms.
                 // WallSO, SpikesSO, PlayerSpawnSO, PlayerWinSO, EnemySO, EnemyFollowSO, MovingPlatformSO, MovingPlatformDeathSO
-                WallSO, SpikesSO, PlayerSpawnSO, PlayerWinSO, EnemySO, EnemyFollowSO
+                this.WallSO, this.SpikesSO, this.PlayerSpawnSO, this.PlayerWinSO, this.EnemySO, this.EnemyFollowSO
             };
```

Comment:
@Bolrik you need to change your IDE settings to stop them doing `this` :stuck_out_tongue:

Reply:
Sorry, not sorry. :D
'This' is a standard in our company and it would be difficult for me to get rid of it. But i will try to stop to use my auto formating rules as much as i can. :x Sorry!